client/src/components/AvatarDisplay/AvatarDisplay.tsx

```diff
@@ -7,6 +7,7 @@ interface Props {
 }
 
 const AvatarDisplay = ({ user }: Props): JSX.Element => {
+  console.log('avatar display here');
```

Comment:
@moffatethan might not like, leaving bug logs and comments in the code. If you could try removing them before you push